src/main/java/com/jagrosh/jmusicbot/commands/owner/PlaylistCmd.java

```diff
@@ -72,7 +72,16 @@ public MakelistCmd()
         protected void execute(CommandEvent event) 
         {
             String pname = event.getArgs().replaceAll("\\s+", "_");
-            if(bot.getPlaylistLoader().getPlaylist(pname)==null)
+            if(pname == null || pname.isEmpty()) 
+            {
+                event.reply(event.getClient().getError() + " Please provide a name for playlist!");
+                
+                StringBuilder builder = new StringBuilder(event.getClient().getWarning() + " Playlist Make Command:\n");
+                builder.append("\n`").append(event.getClient().getPrefix()).append(name)
+                        .append(" <name>` - creates a new, empty playlist with given name");
```

Comment:
I personally believe that this is too excessive.

Reply:
I think so too :) Just wanted to match with the hints that were given if I just call the `playlist` command with no arguments. I'll get rid of it.